cookbook/deployment/configure_logging_links.py

```diff
@@ -60,6 +60,23 @@
               - "https://some-other-source/home?region=us-east-1#logEventViewer:group=/flyte-production/kubernetes;stream=var.log.containers.{{.podName}}_{{.namespace}}_{{.containerName}}-{{.containerId}}.log"
             messageFormat: "json" # "unknown" | "csv" | "json"
 
+.. tip::
+   Since helm chart uses same templating for args (like ``{{ }}``), after compiling the chart, helm replaces flyte log link templates too. To avoid this, use escaped templating for flyte logs in the helm chart.
```

Comment:
```suggestion
   Since helm chart uses the same templating syntax for args (like ``{{ }}``) compiling the chart results in helm replacing Flyte log link templates as well. To avoid this, use escaped templating for Flyte logs in the helm chart. this will ensure that Flyte log link templates remain in place during helm chart compilation.
```